lib/pow_assent/ecto/schema.ex

```diff
@@ -55,10 +51,32 @@ defmodule PowAssent.Ecto.Schema do
         unquote(__MODULE__).changeset(user_or_changeset, user_identity, attrs, user_id_attrs, @pow_config)
       end
 
+      unquote(__MODULE__).__has_many__()
+
       defoverridable unquote(__MODULE__)
     end
   end
 
+  @doc false
+  defmacro __has_many__() do
+    quote do
+      @pow_assocs {:has_many, :user_identities, unquote(__MODULE__).__user_identities_module__(__MODULE__), foreign_key: :user_id, on_delete: :delete_all}
+    end
+  end
+
+  @doc false
+  def __user_identities_module__(module) do
+    [_schema, _context | rest] =
```

Comment:
Should be refactored to handle non-context modules like `MyApp.User`. It's probably better if context app is extracted in a different way. Also this should have a test for overridden assocs with non existent fallback user identities module.